app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/learning_ui/LearningTestRepresentation.kt

```diff
@@ -0,0 +1,69 @@
+package com.github.HumanLearning2021.HumanLearningApp.learning_ui
+import android.content.Intent
+import android.widget.ImageView
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry.getInstrumentation
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiObject
+import androidx.test.uiautomator.UiSelector
+import com.github.HumanLearning2021.HumanLearningApp.R
+import com.github.HumanLearning2021.HumanLearningApp.view.LearningActivity
+import com.github.HumanLearning2021.HumanLearningApp.view.LearningMode
+import com.github.HumanLearning2021.HumanLearningApp.view.LearningSettingsActivity
+import com.schibsted.spain.barista.assertion.BaristaImageViewAssertions.assertHasDrawable
+import org.hamcrest.CoreMatchers.*
+import org.junit.Assert.assertThat
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class LearningTestRepresentation {
+
+    @get:Rule
+    val activityScenarioRule: ActivityScenarioRule<LearningActivity> = ActivityScenarioRule(
+        Intent(
+            ApplicationProvider.getApplicationContext(),
+            LearningActivity::class.java
+        ).putExtra(LearningSettingsActivity.EXTRA_LEARNING_MODE, LearningMode.REPRESENTATION))
+
+
+
+    val NUMBER_OF_CATEGORIES = 3
+    val CATEGORY_VIEW_CLASS_NAME = "android.widget.ImageView"
+    val NUMBER_OF_DRAG_STEPS = 10
+
+    lateinit var mDevice: UiDevice
+    @Before
+    fun init() {
+        mDevice = UiDevice.getInstance(getInstrumentation())
+    }
+
+    @Test
+    fun imageToClassifyIsNotRepresentativePicture(){
+        // TODO: couldn't yet find a way to test whether the picture held by the view was part of one set and not another
+    }
+
+    @Test
+    fun targetImagesAreRepresentativePictures(){
+        // TODO: couldn't yet find a way to test whether the picture held by the view was part of one set and not another
+    }
```

Comment:
I don't know if this is really a problem, tell me what you think but we are already calling the "representativePictures" the pictures who represent a category. In those tests we are not talking about the same types of "representative pictures" right?

Reply:
Yes we are. I wanted to test that only RepresentativePictures were in the target slot, and only dataset pictures were in the image to sort slot

Reply:
okay I see my bad.